backend/app/Http/Controllers/StackController.php

```diff
@@ -12,20 +12,31 @@ class StackController extends Controller
 {
     public function addStack(StackRequest $request): JsonResponse
     {
-        $data = $request->all();
+        $name = $request->name;
+        $user_id = $request->user["id"];
```

Comment:
what's happening here?

Reply:
$name is getting the stack name to be added. 

$user is getting the logged-in user_id

Reply:
oh.. is this what you achieved with this line? great but ..........???????

Reply:
???

Reply:
never mind